dockerfiles/Dockerfile.openvino-rhel8

```diff
@@ -27,20 +27,6 @@ RUN yum install -y yum-utils autoconf automake libtool unzip udev wget zlib-deve
     wget https://www.python.org/ftp/python/3.8.9/Python-3.8.9.tgz && tar xvf Python-3.8.9.tgz && rm -rf Python-3.8.9.tgz && \
     cd Python-3.8*/ && ./configure && make && make install && \
     cd ../ &&  mkdir -p /usr/bin/Python38 && ln -s Python-3.8.9/ /usr/bin/Python38 && ln -s /usr/bin/pip3 /usr/bin/pip && \
-# libusb1.0.22
```

Comment:
Please dont make changes in dockerfiles. We no longer maintain them and any change in them trigger a long dicussion with msft